src/cache/inmemory/__tests__/entityStore.ts

```diff
@@ -2037,4 +2037,101 @@ describe('EntityStore', () => {
       },
     });
   });
+
+  it("supports toReference(id)", () => {
+    const cache = new InMemoryCache({
+      typePolicies: {
+        Book: {
+          fields: {
+            favorited(_, { readField, toReference }) {
+              const rootQueryRef = toReference("ROOT_QUERY");
+              expect(rootQueryRef).toEqual(makeReference("ROOT_QUERY"));
+              const favoritedBooks = readField<[]>("favoritedBooks", rootQueryRef);
```

Comment:
```suggestion
              const favoritedBooks = readField<Reference[]>("favoritedBooks", rootQueryRef);
```
I believe the `[]` type is an empty tuple type, which is something you rarely see.